packages/plugin-essentials/sources/commands/config.ts

```diff
@@ -31,7 +31,19 @@ export default (clipanion: any, pluginConfiguration: PluginConfiguration) => cli
   )
 
   .action(async ({cwd, stdout, verbose, why, json}: {cwd: string, stdout: Writable, verbose: boolean, why: boolean, json: boolean}) => {
-    const configuration = await Configuration.find(cwd, pluginConfiguration);
+    const configuration = await Configuration.find(cwd, pluginConfiguration, {
+      strict: false,
+    });
+
+    if (configuration.invalid.size > 0 && !json) {
+      await StreamReport.start({configuration, stdout, footer: false}, async report => {
+        for (const key of configuration.invalid) {
+          report.reportError(MessageName.INVALID_CONFIGURATION_KEY, `Invalid configuration key "${key}"`);
```

Comment:
Would it make sense to report the `source` too?

Reply:
Yep good idea 👍

Reply:
I've made the `invalid` property a map (rather than use the regular `sources`), since I'd like invalid keys not to change the state of the `Configuration` apart from the `invalid` special property.

Reply:
Makes sense 👍